.travis.yml

```diff
@@ -19,10 +19,14 @@ sudo: false
 language: python
 
 python:
-  - "2.7"
   - "3.5"
```

Comment:
moderate: remove 3.5 too

Reply:
Same as in https://github.com/inveniosoftware/idutils/pull/57, this needs clarification.